src/app/create/codebases/codebases.component.html

```diff
@@ -1,3 +1,5 @@
 <div id="codebases" class="container content">
-  <h1>TODO</h1>
+  <!-- can someone please show me how to make this be a relative url for the current component's directory?
+       I really just want to use 'codebases.png', but I've been unable to figure out how to do that -->
+  <img src='/app/create/codebases/codebases.png' style="width:100%;">
```

Comment:
Thanks, @pmuir, for the explanation and the fix. I think this is worthy of some future consideration, because it makes components feel less 'componentized'. I believe I've seen some people write helper methods which augment the component with the ability to resolve its path. I think you then end up with html such as <img [src]="resolvePath('myimage.png')">.